pocs/mount/__init__.py

```diff
@@ -1 +1,73 @@
+from glob import glob
+
 from pocs.mount.mount import AbstractMount  # pragma: no flakes
+from pocs.utils import error
+from pocs.utils import load_module
+from pocs.utils.logger import get_root_logger
+
+
+def create_mount_from_config(config, mount_info=None, earth_location=None, ignore_local_config=False, simulator=None):
+    """Creates a mount object.
+
+     Details for the creation of the mount object are held in the
+     configuration file or can be passed to the method.
+
+     This method ensures that the proper mount type is loaded.
+
+     Args:
+         mount_info (dict):  Configuration items for the mount.
+
+     Returns:
+         pocs.mount:     Returns a sub-class of the mount type
+     """
+
+    logger = get_root_logger()
+
+    if 'mount' not in config:
+        logger.info("No mount in config")
+        return None
+
+    if not earth_location:
```

Comment:
Since we pass the full config we should probably try and build the `earth_location` from the config if we can as well. So check for `None` here and otherwise use `create_location_from_config` and if still can't create a location, then warn and fail.